remappings.txt

```diff
@@ -1,5 +1,12 @@
 ds-test/=lib/forge-std/lib/ds-test/src/
 forge-std/=lib/forge-std/src/
-@openzeppelin/contracts/=lib/openzeppelin-contracts/contracts/
-@openzeppelin/contracts-upgradeable/=lib/openzeppelin-contracts-upgradeable/contracts/
+@oz/contracts/=lib/openzeppelin-contracts/contracts/
```

Comment:
had to rename this to avoid conflicts with the @openzeppelin from `lib/account-abstraction`